Kiss.Bff/ZaakGerichtWerken/Contactverzoeken/ReadContactverzoekenVragenSets.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Text.Json;
+using Kiss.Bff.Beheer.Data;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Kiss.Bff.ZaakGerichtWerken.Contactverzoeken
+{
+    [ApiController]
+    public class ReadContactverzoekenVragenSets : ControllerBase
+    {
+        private readonly BeheerDbContext _db;
+
+        public ReadContactverzoekenVragenSets(BeheerDbContext db)
+        {
+            _db = db;
+        }
+
+        [HttpGet("/api/contactverzoekvragensets")]
+        public async Task<IActionResult> Get(CancellationToken token)
+        {
+            var list = await _db.ContactVerzoekVragenSets.ToListAsync();
```

Comment:
hoe komt er data in deze tabel?

Reply:
moeten we zelf doen want er is nog geen cms met crud

`INSERT INTO "ContactVerzoekVragenSets" ("Naam", "JsonVragen", "AfdelingId")
VALUES
('Afspraak consultatiebureau (tot 4 jaar)', '[
  "Naam Kind",
  "Geboortedatum Kind",
  "Postcode en huisnummer van het kind",
  "Klant heeft al een afspraak: op welke datum?",
  "Klant wil graag afspraak maken: op welke datum?"
]', 'dept-id-1'),

('Afspraak schoolarts (4 t/m 18 jaar)', '[
  "Naam Kind",
  "Geboortedatum Kind",
  "Postcode en huisnummer van het kind",
  "Klant heeft al een afspraak: op welke datum?",
  "Klant wil graag afspraak maken: op welke datum?"
]', 'dept-id-2'),

('WMO', '[
  "Waar gaat de vraag over?",
  "Als anders: vul onderwerp in",
  "BSN klant",
  "Adres klant",
  "Naam WMO-consulent"
]', 'dept-id-3'),

('Vraag over vergunning', '[
  "Postcode en huisnummer van de vergunning",
  "Vergunning nummer"
]', 'dept-id-4'),

('Contactverzoek WI', '[
  "BSN klant",
  "Wanneer wil de klant BIJ VOORKEUR teruggebeld worden"
]', 'dept-id-5');`